src/main/asciidoc/reference/mapping.adoc

```diff
@@ -39,6 +44,15 @@ By creating your own instance, you can register Spring converters to map specifi
 
 You can configure the `MappingR2dbcConverter` as well as `DatabaseClient` and `ConnectionFactory` by using  Java-based metadata. The following example uses Spring's Java-based configuration:
 
+If you set `setForceQuote` of the `R2dbcMappingContext to `true` table and column names derived from classes and properties will be used with database specific quotes.
+This means it is ok to use reserved SQL words like for example `order` in these names.
+This can be done by overriding `r2dbcMappingContext(Optional<NamingStrategy>)` of `AbstractR2dbcConfiguration`.
+Spring Data converts the letter casing of such a name to that form which is also used by the configured database when no quoting is used.
+Therefore, you can use unquoted names when creating tables, as long as you don't use keywords or special characters in your names.
+For databases adhering to the SQL standard with that respect this means, names will be converted to upper case.
```

Comment:
For databases that adhere to the SQL standard , this means that names are converted to upper case.